src/pages/signup/Signup.jsx

```diff
@@ -0,0 +1,104 @@
+import React, {useState} from 'react';
+import { Row, Col, Card } from 'antd';
+import Content from '../../components/signup/content/Content';
+import SignupForm from '../../components/signup/signup/SignupForm';
+import LoginForm from '../../components/signup/login/LoginForm';
+import logo from '../../assets/logo-white.svg';
+import './signup.css';
+import { useRouteMatch } from 'react-router-dom';
+import * as firebase from "firebase/app";
+import "firebase/auth";
+import history from '../../history';
+import client from '../../client';
+import {enterpriseSignin, notify} from '../../utils';
+import store from '../../store';
+import {increment, decrement} from 'automate-redux';
+
+const Signup = () => {
+    let match = useRouteMatch();
+    const googleProvider = new firebase.auth.GoogleAuthProvider();
+    const twitterProvider = new firebase.auth.TwitterAuthProvider();
+    const githubProvider = new firebase.auth.GithubAuthProvider();
+    
+    const handleGoogleSignin = () => {
+        firebase.auth().signInWithPopup(googleProvider).then((userInfo) =>{
```

Comment:
Replace userinfo with res and use the getIdToken method instead of refreshToken everywhere